chrome-extension/plugin/pluginbuilder.ts

```diff
@@ -22,7 +22,7 @@ class PluginState {
 
 export class Plugin {
   domain: string;
-  shortcuts: any[];
+  shortcuts: any[]; // MDS
```

Comment:
re: shortcuts data representation
MDS = macro-data schema (macro-data, rendering panel)
DS = developer schema (in Plugins)

Reply:
ouh, gotcha